osbs/api.py

```diff
@@ -475,7 +475,6 @@ def _do_create_prod_build(self, git_uri, git_ref,
             koji_kerberos_keytab=self.build_conf.get_koji_kerberos_keytab(),
             koji_kerberos_principal=self.build_conf.get_koji_kerberos_principal(),
             architecture=architecture,
-            platform=platform,
```

Comment:
It's a required param for the `create_worker_build` api method which is called by atomic-reactor's `orchestrate_build` plugin: https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/build_orchestrate_build.py#L203